WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -500,36 +504,49 @@ public void onCreate(Bundle savedInstanceState) {
         }
     }
 
-    private void toggleJetpackBannerIfEnabled(boolean forceShow) {
-        if (!isAdded() || !isSearching() || getView() == null) return;
-        final boolean shouldShow = forceShow && mJetpackBrandingUtils.shouldShowJetpackBranding();
-        if (shouldShow) {
-            View jetpackBanner = getView().findViewById(R.id.jetpack_banner);
-            jetpackBanner.setVisibility(View.VISIBLE);
+    private void toggleJetpackBannerIfEnabled(final boolean showIfEnabled, boolean animateOnScroll) {
+        if (!isAdded() || getView() == null || !isSearching()) return;
 
-            if (mJetpackBrandingUtils.shouldShowJetpackPoweredBottomSheet()) {
-                jetpackBanner.setOnClickListener(v -> {
-                    mJetpackBrandingUtils.trackBannerTapped(Screen.READER_SEARCH);
-                    new JetpackPoweredBottomSheetFragment()
-                            .show(getChildFragmentManager(), JetpackPoweredBottomSheetFragment.TAG);
-                });
+        if (animateOnScroll) {
```

Comment:
I believe this line bypasses the `mJetpackBrandingUtils.shouldShowJetpackBranding` check, which makes this banner visible in the Jetpack app, as described here https://github.com/wordpress-mobile/WordPress-Android/pull/17028#issuecomment-1222166253.

Reply:
Nicely spotted, that was indeed the problem. Thank you for fixing this in https://github.com/wordpress-mobile/WordPress-Android/pull/17062

Reply:
This was fixed with: https://github.com/wordpress-mobile/WordPress-Android/pull/17062